bindata/network/multus/multus.yaml

```diff
@@ -630,19 +658,58 @@ spec:
             current-context: whereabouts-context
             EOF
 
-            # Copy the config from ConfigMap to the desired directory
-            cp /etc/whereabouts/config/whereabouts.conf $WHEREABOUTS_GLOBALCONFIG
-            chmod 600 $WHEREABOUTS_GLOBALCONFIG
-
             else
               warn "Doesn't look like we're running in a kubernetes environment (no serviceaccount token)"
             fi
 
+            }
+
+            generateKubeConfig
+
+            # ------------------ end Generate a "kube-config"
+
+            # ----------------- Generate a whereabouts conf
+
+            function generateWhereaboutsConf {
+
+              touch $WHEREABOUTS_CONF_FILE
+              chmod ${KUBECONFIG_MODE:-600} $WHEREABOUTS_CONF_FILE
+              cat > $WHEREABOUTS_CONF_FILE <<EOF
+            {
+              "datastore": "kubernetes",
+              "kubernetes": {
+                "kubeconfig": "${WHEREABOUTS_KUBECONFIG_LITERAL}"
+              },
+              "reconciler_cron_expression": "${WHEREABOUTS_RECONCILER_CRON}"
+            }
+            EOF
+
+            }
+
+            generateWhereaboutsConf
+
+            # ---------------- End generate a whereabouts conf
+
+
+
             # copy whereabouts to the cni bin dir
-            # SKIPPED DUE TO FIPS COPY.
-            # cp -f /whereabouts $CNI_BIN_DIR
+            cp -f /whereabouts $CNI_BIN_DIR
+
+            # ---------------------- end generate a "kube-config".
+
+            # enter sleep/watch loop
+
+            while true; do
+              # Check the md5sum of the service account token and ca.
+                svcaccountsum=$(md5sum $SERVICE_ACCOUNT_TOKEN_PATH | awk '{print $1}')
+                casum=$(md5sum $KUBE_CA_FILE | awk '{print $1}')
+                if [ "$svcaccountsum" != "$LAST_SERVICEACCOUNT_MD5SUM" ] || [ "$casum" != "$LAST_KUBE_CA_FILE_MD5SUM" ]; then
+                  # log "Detected service account or CA file change, regenerating kubeconfig..."
+                  generateKubeConfig
+                fi
 
-            # ---------------------- end Generate a "kube-config".
+                sleep 1
+              done
```

Comment:
the above is money, I know this is the thing we need, but... also check funky indents here on `done`